README.md

```diff
@@ -1 +0,0 @@
-# microflix
```

Comment:
You made a great effort on this project. However, it has one thing that needs to be completed to make your project approved.

Kindly assign the person to each task equally. 

WRONG:

<img width="358" alt="Screen Shot 2022-08-02 at 00 33 27" src="https://user-images.githubusercontent.com/69011963/182235602-60e8e575-37eb-459b-8f15-a31cec24cb7f.png">

CORRECT:

<img width="372" alt="assigned_card" src="https://user-images.githubusercontent.com/69011963/182235693-06fa61d9-bb2d-4e10-ad33-b0b61e0d86f1.png">

Reply:
In a group task, you can assign both of you.